videointelligence/tests/unit/gapic/v1beta2/test_video_intelligence_service_client_v1beta2.py

```diff
@@ -11,67 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""Unit tests."""
 
-import mock
-import unittest
-
-from google.gax import errors
 from google.rpc import status_pb2
 
 from google.cloud import videointelligence_v1beta2
 from google.cloud.videointelligence_v1beta2.proto import video_intelligence_pb2
 from google.longrunning import operations_pb2
 
 
-class CustomException(Exception):
-    pass
+class UnaryUnaryMultiCallableStub(object):
```

Comment:
@lukesneeringer @dhermes it may make sense to put a full, fleshed out gRPC channel stub (with method stubs) in `api_core` for the benefit of testing all gapics and people who want to test their own gRPC client. WDYT?